<?xml version="1.0"?>


<launch>

  <include file="$(find ucl_drone)/launch/components/global_params.xml" />
  <include file="$(find ucl_drone)/launch/components/gui.xml" />
  <include file="$(find ucl_drone)/launch/components/slam.xml" />
  <include file="$(find ucl_drone)/launch/components/driver.xml" />

  <node name="ucl_drone_manual_pose_estimation" pkg="ucl_drone" type="manual_pose_estimation" output="screen" />

  <!-- overriding default values of included files -->
  <param name="ucl_drone_mapping_node/sonar_unavailable"    value="true" />

  <param name="ucl_drone_computer_vision/autonomy_unavailable" value="true" />
</launch>
